Makefile.release

```diff
@@ -44,20 +44,17 @@ EMPTY:=
 SPACE:=$(EMPTY) $(EMPTY)
 COMMA:=$(EMPTY),$(EMPTY)
 
-ifeq (, $(shell which curl))
+ifeq (, $(shell command -v curl))
     $(error "No curl in $$PATH, please install")
 endif
-ifeq (, $(shell which manifest-tool))
-    $(error "No manifest-tool in $$PATH, please install")
-endif
 
 DOCKER:=
 NAME:=coredns
 VERSION:=$(shell grep 'CoreVersion' coremain/version.go | awk '{ print $$3 }' | tr -d '"')
 GITHUB:=coredns
 DOCKER_IMAGE_NAME:=$(DOCKER)/$(NAME)
-# mips is not in LINUX_ARCH because it's not supported by the manifest-tool.
-LINUX_ARCH:=amd64 arm arm64 ppc64le s390x
+DOCKER_CLI:=docker
```

Comment:
why this? It's even long to type?

Reply:
Extracting it, so at some point folks can replace it with buildah or similar alternatives.